service/popm/popm.go

```diff
@@ -355,7 +393,12 @@ func (m *Miner) mineKeystone(ctx context.Context, ks *hemi.L2Keystone) error {
 
 	// Estimate BTC fees.
 	txLen := 285 // XXX: for now all transactions are the same size
-	feePerKB := 1024 * m.cfg.StaticFee
+	url := "https://mempool.space/testnet/api/v1/fees/recommended"
```

Comment:
I don't think we want to rely on `mempool.space` for fee estimation, we can use the existing electrumx connection and perhaps the [fee estimation](https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-estimatefee) method.